### DISCUSSION TITLE
Natural ventilation

### DISCUSSION THREAD

**Discussion by ShehabGamrah** [firemodels/fds#13476] [General]
I have a question with regards to natural openings and ventilation. Consider I model a car park or building as empty from inside with equally sized natural openings and on the same height, on opposite wall. Also I don't consider wind. How would the velocity pattern look like, if both openings are specified as the open condition.

**Comment by drjfloyd** [comment]
It would depend upon the internal geometry of the car park, the fire location, any mechanical ventilation on the inside, the fire size, the size of the openings including the top and bottom height, .....

**Reply by ShehabGamrah** [reply]
Assume the car park is symmetrical and there is no fire just want to model the airflow with the openings same size.

**Reply by drjfloyd** [reply]
If you have no fire and no wind, what is going to cause air flow?
